mazda/gps/nematode/include/nmeaparse/Event.h

```diff
@@ -142,7 +142,7 @@ namespace nmea {
 		Event() : enabled(true)
 		{}
 
-		virtual ~Event() 
```

Comment:
The files in the nematode folder are a 3rd party lib, we probably shouldn't reformat them incase we want to merge a new  version later, it comes from here: https://github.com/ckgt/NemaTode

Reply:
ok good point